package.json

```diff
@@ -39,7 +39,7 @@
     "lint": "nx workspace-lint && nx run-many --target=lint --max-warnings=0 --all --parallel",
     "lint:fix": "nx workspace-lint && nx run-many --target=lint --max-warnings=0 --fix --all --parallel",
     "ng": "nx",
-    "ngcc": "ngcc --properties es2015 module browser main",
+    "ngcc": "ngcc --properties es2020 module browser main",
```

Comment:
Have people generally signed off on us building for es2020 instead of es2015? :) I don't remember having this discussion with anyone, so i am interested in what people think.

Reply:
I this specific case it's used for compiling dependencies to es2020, not apps. The version for apps is set in `tsconfig.base.json` which is still set to `es2015`.

Generally, Angular itself changed the target from `es2015` to `es2020` in v14 (which I haven't set in this PR). Maybe we should.

As for our apps, DataHub supports only the latest versions of evergreen browsers until we want another strategy. Have you discussed browser support for Energy Origin?

Reply:
We have not discussed browser support no 😄 But thanks! Yet another reason why I think we should move on the "guild/chapter" idea! :)

From my point of view we should upgrade it all to es2020.